script.js

```diff
@@ -11,4 +13,338 @@ hamburger.addEventListener('click', toggleMenu);
+  popupSourceLinkDesk.appendChild(popupSourceLinkIconDesk);
+
+  const popupTagsMob = document.createElement('ul');
+  popupTagsMob.classList.add('popup-items', 'mobile');
+  popupWindow.appendChild(popupTagsMob);
+
+  const popupTagsDesk = document.createElement('ul');
+  popupTagsDesk.classList.add('popup-items', 'desktop');
+  popupWindow.appendChild(popupTagsDesk);
+
+  for (let j = 0; j < project.languages.length; j += 1) {
+    const popupTag = document.createElement('li');
+    popupTag.innerHTML = project.languages[j];
+    popupTagsMob.appendChild(popupTag);
+  }
+  for (let j = 0; j < project.popupLanguages.length; j += 1) {
+    const popupTag = document.createElement('li');
+    popupTag.innerHTML = project.popupLanguages[j];
+    popupTagsDesk.appendChild(popupTag);
+  }
+
+  const popupDescription = document.createElement('p');
+  popupDescription.classList.add('popup-text');
+  popupDescription.innerHTML = project.description;
+  popupWindow.appendChild(popupDescription);
+
+  const popupButtonsContainerMobile = document.createElement('div');
+  popupButtonsContainerMobile.classList.add('popup-buttons', 'mobile');
+  popupWindow.appendChild(popupButtonsContainerMobile);
+
+  const popupLiveLink = document.createElement('button');
+  popupLiveLink.classList.add('button');
+  popupLiveLink.innerHTML = 'See Live';
+  popupButtonsContainerMobile.appendChild(popupLiveLink);
+
+  const popupLiveLinkIcon = document.createElement('img');
+  popupLiveLinkIcon.setAttribute('src', 'images/see-live-icon.png');
+  popupLiveLink.appendChild(popupLiveLinkIcon);
+
+  const popupSourceLink = document.createElement('button');
+  popupSourceLink.classList.add('button');
+  popupSourceLink.innerHTML = 'See Source';
+  popupButtonsContainerMobile.appendChild(popupSourceLink);
+
+  const popupSourceLinkIcon = document.createElement('img');
+  popupSourceLinkIcon.setAttribute(
+    'src',
+    'images/See-source.png',
+  );
+  popupSourceLinkIcon.setAttribute('alt', 'Source Link Icon');
+  popupSourceLink.appendChild(popupSourceLinkIcon);
+
+  x.appendChild(popupWindow);
+
+  document.body.appendChild(popupWindow);
+
+  popupCloseButton.addEventListener('click', () => {
+    overlay.remove();
+    popupWindow.remove();
+  });
+
+  popupLiveLink.addEventListener('click', () => {
+    window.open(project.liveLink);
+  });
+
+  popupSourceLink.addEventListener('click', () => {
+    window.open(project.sourceLink);
+  });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const projectButtons = document.querySelectorAll('#project-button');
+
+  for (let i = 0; i < projectButtons.length; i += 1) {
+    ((index) => {
+      projectButtons[index].addEventListener('click', () => {
+        createPopup(index);
+      });
+    })(i);
+  }
+});
+
+const form = document.getElementById('form');
+
+function invalidMessage() {
+  // eslint-disable-next-line no-alert
+  window.alert('please, use only lowercase');
+}
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure the error message can show up around the input fields or just around the submit button.
- [ ] As stated in the project requirements, kindly ensure the form `"submit"` or the submit button/input `"click"` event is used to attach the function checking the email format.
- [ ] I would recommend you don't use the alert function but rather render the error message to the DOM around the input fields just like stated in the project requirements.